source/administration/command-line-tools.rst

```diff
@@ -35,18 +35,37 @@ From the directory where the Mattermost platform is installed, a
     :backlinks: top
     :local:
 
-Mattermost 3.6 and later
-^^^^^^^^^^^^^^^^^^^^^^^^
+Using the CLI
+^^^^^^^^^^^^^
 
-The new CLI tool is supported in Mattermost 3.6 and later. To see available commands in the old CLI tool, see `Mattermost 3.5 and earlier`_.
+To run the CLI commands, you must be in the directory that contains the Mattermost executable. On a default install of Mattermost, the directory is ``/opt/mattermost/bin``. The name of the executable is ``platform``.
 
-Typing ``sudo ./platform help`` and ``sudo ./platform help {command}`` returns help documentation for the CLI tool or any CLI command in particular.
+**For example, to get the Mattermost version on a default installation of Mattermost:**
 
-To return the help documentation in GitLab omnibus, type
+  .. code-block:: bash
 
-    .. code-block:: none
+    cd /opt/mattermost/bin
+    sudo ./platform version
+
+Using the CLI on GitLab Omnibus
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+On GitLab Omnibus, you must be in the following directory: ``/opt/gitlab/embedded/service``. The name of the executable is ``mattermost``. Also, you must run the commands as the user *mattermost* and specify the location of the configuration file.
```

Comment:
For my installation which as far as I know is standard in every way, I have to be in the directory: ``/opt/gitlab/embedded/service/mattermost``, to avoid the nil pointer runtime errors
(Missing the final mattermost directory)

The actual mattermost executable is still located at ``/opt/gitlab/embedded/bin/mattermost``, but it doesn't work unless you run it from the directory specified above.